renovate.json

```diff
@@ -12,21 +13,26 @@
 	"internalChecksFilter": "strict",
 	"packageRules": [
 		{
-			"description": "Wait 3 days before creating a npm update PR",
-			"matchDatasources": ["npm"],
-			"stabilityDays": 3
-		},
-		{
-			"description": "Automerge ESLint and Prettier updates",
+			"description": [
+				"Group all non-major pnpm development dependencies together, get updates once a month"
+			],
+			"matchManagers": ["npm"],
 			"matchDepTypes": ["devDependencies"],
```

Comment:
This is the key part: `matchDepTypes=devDependencies` makes it so this only groups `devDependencies` in the `package.json` file.

Our framework stack (Next.js and React) is in the `dependencies` part of the `package.json` file:

```json
	"dependencies": {
		"@chakra-ui/react": "2.2.8",
		"@emotion/react": "11.10.0",
		"@emotion/styled": "11.10.0",
		"@octokit/auth-callback": "3.0.0",
		"@octokit/rest": "19.0.3",
		"@sentry/nextjs": "7.10.0",
		"@tanstack/react-query": "4.1.0",
		"@tanstack/react-query-devtools": "4.0.10",
		"downshift": "6.1.9",
		"framer-motion": "7.1.0",
		"lodash-es": "4.17.21",
		"next": "12.2.5",
		"next-seo": "5.5.0",
		"nookies": "2.5.2",
		"react": "18.2.0",
		"react-dom": "18.2.0",
		"react-icons": "4.4.0",
		"rehype-highlight": "5.0.2",
		"rehype-react": "7.1.1",
		"remark-gfm": "3.0.1",
		"remark-github": "11.2.4",
		"remark-parse": "10.0.1",
		"remark-rehype": "10.1.0",
		"remark-stringify": "10.0.2",
		"semver": "7.3.7",
		"unified": "10.1.2"
	},
```

So these `dependencies` are _not_ affected by the new grouping. We'll get separate PRs for each of these `dependencies` when they're three days old.

Reply:
Great! Thanks for the clarification.